schemas/helper.py

```diff
@@ -12,7 +12,7 @@ async def send_text_message(user: UserInfo, message: str):
             headers=HEADER,
             json={
                 "recipient": {"id": user.recipient_id},
-                "message": generateTextMessage(message)
+                "message": message
```

Comment:
I think here is the problem, i wrapped it up as a function because it can return a dict object instead of a pure string,